setup.cfg

```diff
@@ -12,12 +12,11 @@ tag_prefix =
 parentdir_prefix = modin-
 
 [tool:pytest]
-addopts = --cov-config=setup.cfg --cov=modin --cov-append --cov-report=
+addopts = --cov-config=setup.cfg --cov=modin --cov-append --cov-report= -m "not exclude_by_default"
```

Comment:
why do we need to add an extra option here? isn't `-m "not exclude..."` is identical to passing nothing?

Reply:
Oh, I completely forgot about this change and confused you. In fact, the default value for the marker is `True`.